packages/valory/contracts/service_staking_token/contract.py

```diff
@@ -112,3 +112,26 @@ def get_next_checkpoint_ts(
         contract = cls.get_instance(ledger_api, contract_address)
         ts = contract.functions.getNextRewardCheckpointTimestamp().call()
         return dict(data=ts)
+
+    @classmethod
+    def get_liveness_period(
+        cls,
+        ledger_api: LedgerApi,
+        contract_address: str,
+    ) -> JSONLike:
+        """Retrieve the liveness period."""
+        contract = cls.get_instance(ledger_api, contract_address)
+        liveness_period = contract.functions.livenessPeriod().call()
+        return dict(data=liveness_period)
+
+    @classmethod
+    def get_service_info(
+        cls,
+        ledger_api: LedgerApi,
+        contract_address: str,
+        service_id: int,
+    ) -> JSONLike:
+        """Retrieve the service info for a service."""
+        contract = cls.get_instance(ledger_api, contract_address)
+        info = contract.functions.mapServiceInfo(service_id).call()
```

Comment:
you are calling the interface method get_service_info, but you are calling "mapServiceInfo" instead of "getServiceInfo" in the contract. Is this correct?

Reply:
I would say, either use getServiceInfo, or rename to map_service_info, to be consistent with the contract interface.

Reply:
https://github.com/valory-xyz/autonolas-registries/blob/main/contracts/staking/ServiceStakingBase.sol#L594
Its the same